services/src/main/java/org/exoplatform/news/queryBuilder/NewsQueryBuilder.java

```diff
@@ -54,7 +54,9 @@ public StringBuilder buildQuery(NewsFilter filter) throws Exception {
         }
         if (filter.getSearchText() != null && !filter.getSearchText().equals("")) {
           String escapedQuoteSearchText = filter.getSearchText().replace("'", "''").replace("\"", "\"\"");
-          sqlQuery.append("CONTAINS(.,'").append(escapedQuoteSearchText).append("')");
+          int index = escapedQuoteSearchText.indexOf("~0.6");
+          String searchTextTerm = escapedQuoteSearchText.substring(0, index);
+          sqlQuery.append("(CONTAINS(.,'").append(escapedQuoteSearchText).append("') OR (exo:body LIKE '%").append(searchTextTerm).append("%'))");
```

Comment:
the directive Contains will look for the exact word, when the word is put inside a HTML tag with no spaces before and after it won't be returned. 
You can remove the usage of CONTAINS but you have to include other properties of News in the search query exo:summary, exo:title.
A better solution would be to use Elastic search instead of searching the JCR. Could you evaluate the complexity of such change ?